### DISCUSSION TITLE
Question: Is there any way I can customize the DagRun endpoints

### DISCUSSION THREAD

**Discussion by bfan4** [apache/airflow#14041] [Q&A] [answered]
### Background: 

Airflow updated to 2.0.0 and one of the new features is the stable Restful API, I'm using the `/api/v1/dags/{dag_id}/dagRuns` API to trigger a DAG, I have some parameters in the request body:
```
{
    "conf": {
        "book_id": "12"
    }
}
```
if I rename the "conf" to "config" or make the request body empty, I got `400 BAD REQUEST`, that's OK, I know airflow will check the request body. But if I want `book_id` to be a required field and missing the `book_id` will return `400 BAD REQUEST` how can I implement that?

**Comment by ashb** [comment, marked as answer]
Right now that isn't possible, at least not with some heavy hacking.

Could you open a feature request issue for this?

**Reply by bfan4** [reply]
Thanks for your response! Will do that.
